Weather/Services/UserLocations/UserLocationsService.swift

```diff
@@ -0,0 +1,10 @@
+import Foundation
+
+
+protocol UserLocationsService {
```

Comment:
Can we change the protocol name to something like:

LocationStoreService

as otherwise it looks like something that relates to the actual user location vs a store for locations that user choose :)